src/pages/Admin/Venue/DetailsForm.tsx

```diff
@@ -742,60 +715,28 @@ const DetailsFormLeft: React.FC<DetailsFormLeftProps> = ({
   const renderShowReactions = () => (
     <div className="toggle-room">
       <h4 className="italic input-header">Show reactions</h4>
-      <label id="showReactions" className="switch">
-        <input
-          type="checkbox"
-          id="showReactions"
-          name="showReactions"
-          ref={register}
-        />
-        <span className="slider round"></span>
-      </label>
+      <Toggler name="showReactions" forwardedRef={register} />
     </div>
   );
 
   const renderShowShoutouts = () => (
     <div className="toggle-room">
       <h4 className="italic input-header">Show shoutouts</h4>
-      <label id="showShoutouts" className="switch">
-        <input
-          type="checkbox"
-          id="showShoutouts"
-          name="showShoutouts"
-          ref={register}
-        />
-        <span className="slider round"></span>
-      </label>
+      <Toggler name="showShoutouts" forwardedRef={register} />
     </div>
   );
 
   const renderShowRangersToggle = () => (
     <div className="toggle-room">
       <h4 className="italic input-header">Show Rangers support</h4>
-      <label id="showRangers" className="switch">
-        <input
-          type="checkbox"
-          id="showRangers"
-          name="showRangers"
-          ref={register}
-        />
-        <span className="slider round"></span>
-      </label>
+      <Toggler name="showRangers" forwardedRef={register} />
     </div>
   );
 
   const renderRestrictDOBToggle = () => (
```

Comment:
```suggestion
  // @debt pass the header into Toggler's 'label' prop instead of being external like this
  const renderRestrictDOBToggle = () => (
```